build/package/.goreleaser.yml

```diff
@@ -47,7 +47,7 @@ gomod: # https://goreleaser.com/customization/verifiable_builds/
   # This will only be effective if running against a tag. Snapshots will ignore
   # this setting.
   # Notice: for this to work your `build.main` must be a package, not a `.go` file.
-  proxy: false # it will be enabled in CLOUDP-228737
+  proxy: true
```

Comment:
@gssbzn SBOM works but we will revert this once we move from `atlascli/vX.Y.Z` to `vX.Y.Z`, do you think it is still worth committing?

Reply:
I don't think we should rollback the versioning name, it opens too many questions for things like the update notifier that I think if this works we should live with it